### PR TITLE
Single server storage note

### PR DESCRIPTION
**Pull request by dansisson**

### CODE REVIEW

**Comment by dansisson** [comment]
If this PR needs approval, or if the commits need to be pulled into the private repo, contact @alkohli. Thanks.

**Comment by Court72** [comment]
@alkohli

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/concepts/system-requirements-23h2.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ Here are the Azure requirements for your Azure Stack HCI cluster:
 
 Before you begin, make sure that the physical server and storage hardware used to deploy an Azure Stack HCI cluster meets the following requirements:
 
+> [!NOTE]
+> Single servers must use only a single drive type: Non-volatile Memory Express (NVMe) or Solid-State (SSD) drives.
+
```
Comment (alkohli) [inline azure-stack/hci/concepts/system-requirements-23h2.md:46]: The placement of the above note should be in the table in the following row: 

|Data drives|At least 2 disks with a minimum capacity of 500 GB (SSD or HDD).<br>Single servers must use only a single drive type: Non-volatile Memory Express (NVMe) or Solid-State (SSD) drives.|

File: azure-stack/hci/plan/single-server-deployment.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ In this article, you'll learn about the single-server storage network reference
 
 For information on other network patterns, see [Azure Stack HCI network deployment patterns](choose-network-pattern.md).
 
+> [!NOTE]
+> Single servers must use only a single drive type: Non-volatile Memory Express (NVMe) or Solid-State (SSD) drives.
+
```
Comment (alkohli) [inline azure-stack/hci/plan/single-server-deployment.md:23]: Move the note to the bottom of the Scenarios section.

**Review by alkohli — COMMENTED** [review]
Dan - if you can fix these, I will sign off this PR.

**Comment by alkohli** [comment]
#sign-off Looks good. Ready to merge.

**Pull request merged**